DTS-STN/Alpha-Site
# Update required fields

# Description

[Add proper invalid attribute to required fields on error](https://jira-dev.bdm-dev.dts-stn.com/browse/SCL-574)

Set an area-invalid attribute to the inputs containing errors when the form is submitted with errors.

[Add proper invalid attribute to required fields on error](https://jira-dev.bdm-dev.dts-stn.com/browse/SCL-574)

Add aria-hidden the * before the label and the (required) after the label, and change the HTML for the inputs from required="required" to aria-required="true".

## Test Instructions

1. Run the code locally
2. Go to the signup form
3. Check the required fields to make sure the * before the label and the (required) after the label have an aria-hidden attribute
4. Then check to see if the inputs have the aria-required="true"
5. After that, submit the form without filling it
6. Check to see if the required fields have an area-invalid attribute set to the inputs with errors

## Checklist


## Product and Sprint Backlog

[Jira](https://jira-dev.bdm-dev.dts-stn.com/secure/RapidBoard.jspa?rapidView=96&projectKey=SCL)
